tests/Feature/Api/ApplicantSkillsControllerTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Tests\Feature\Api;
+
+use App\Models\Applicant;
+use App\Models\Skill;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class ApplicantSkillsControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testGetApplicantSkills(): void
+    {
+        $applicant = factory(Applicant::class)->create();
+        $skills = Skill::inRandomOrder()->limit(5)->get();
+        $applicant->skills()->sync($skills->pluck('id'));
+
+        // Assert this is auth protected
+        $guestResponse = $this->json('get', route('api.v1.applicant.skills.index', $applicant));
+        $guestResponse->assertForbidden();
+
+        $response = $this->actingAs($applicant->user)->json('get', route('api.v1.applicant.skills.index', $applicant));
+        $response->assertOk();
+        $response->assertJson([
+            'skill_ids' => $skills->sortBy('id')->pluck('id')->all()
+        ]);
+    }
+
+    public function testUpdateApplicantSkills(): void
+    {
+        $applicant = factory(Applicant::class)->create();
+
+        // Assert this is auth protected
+        $guestResponse = $this->json(
+            'put',
+            route('api.v1.applicant.skills.update', $applicant),
+            ['skill_ids' => []]
+        );
+        $guestResponse->assertForbidden();
```

Comment:
Running this test locally failed for me: `Response status code [419] is not a forbidden status code. Failed asserting that false is true.`. Not sure if this is an issue with the test itself though since it passed through the PHP Unit test on GitHub actions.

Reply:
Tests are passing on my end ✅